src/constants.py

```diff
@@ -32,6 +32,12 @@
 Useful if you want to debug two players and you only have one controller.
 """
 
+PLAYER_ON_PATROL_LOOP = None
+"""
+If set to a player index, 0 to 3, then that player will be moved on autopilot in
+a loop around the map, giving you a moving target to shoot at while testing.
+"""
+
```

Comment:
Change the player index to be the player number. for example use 2 for player2 instead of 1